docs/gradle-getting-started.md

```diff
@@ -64,10 +64,12 @@ Execute the following commands to clone the sample application:
 :::>- $ git clone https://github.com/heroku/gradle-getting-started.git
 :::>- $ cd gradle-getting-started
 :::-- $ git fetch
-:::-- $ git merge origin/master
+:::-- $ git checkout -t origin/spring
```

Comment:
Why would we ask the readers to do a fetch and checkout of a branch?

Reply:
(otherwise, all looking good - though I didn't run through the instructions.  If you'd like me to, give me a ping and I'll do so)

Reply:
(oh is this just temporary and will automagically fix when merged?)

Reply:
Yea, sorry. the `:::--` means that rundoc will execute it, but it won't show up in the resulting documentation. I should actually remove it from the PR